artifacts/artifact_funcs.jl

```diff
@@ -59,3 +60,26 @@ function pr_obs_data_path()
     )
     return AW.get_data_folder(pr_obs_data)
 end
+
+"""
+    artifact_data(datapath, datafile, date0=nothing, time_start=nothing, time_end=nothing)
+
+Truncates given data set, and constructs a new dataset containing only the dates that are used in the simulation
+"""
+function artifact_data(datapath_full, name)
+    datafile_truncated = joinpath(datapath_full, string(name, ".nc"))
+    return datafile_truncated
+end
+
+function artifact_data(datapath_full, name, datapath_trunc, date0, time_start, time_end, comms_ctx)
+    datafile = joinpath(datapath_full, string(name, ".nc"))
+    println(date0) 
+    println(time_start)
+    println(time_end) 
+    println(datafile) 
+    println(datapath_trunc) 
+    println(name) 
+    println(comms_ctx)
+    datafile_truncated = Regridder.truncate_dataset(date0, time_start, time_end, datafile, datapath_trunc, name, comms_ctx)
```

Comment:
it would be nice for `artifact_data` and `truncate_dataset` to have arguments in the same order so we don't have to reorder them between calls to these functions